installer/installer.py

```diff
@@ -32,15 +32,22 @@
 DEFAULT_CHEQD_USER = "cheqd"
 DEFAULT_BINARY_NAME = "cheqd-noded"
 DEFAULT_COSMOVISOR_BINARY_NAME = "cosmovisor"
-DEFAULT_CHAINS = ['testnet', 'mainnet']
-DEFAULT_CHAIN = "mainnet"
+DEFAULT_CHAINS = ['cheqd-testnet-4', 'cheqd-mainnet-1']
```

Comment:
Let's discuss whether we ask them for just mainnet/testnet and set chain ID based on that.

Reply:
Done! ✅